includes/class-wc-google-gtag-js.php

```diff
@@ -286,23 +278,26 @@ public static function load_analytics( $order = false ) {
 		$track_404_enabled = '';
 		if ( 'yes' === self::get( 'ga_404_tracking_enabled' ) && is_404() ) {
 			// See https://developers.google.com/analytics/devguides/collection/gtagjs/events for reference
-			$track_404_enabled = self::tracker_var() . "( 'event', '404_not_found', { 'event_category':'error', 'event_label':'page: ' + document.location.pathname + document.location.search + ' referrer: ' + document.referrer });";
+			$track_404_enabled = "gtag( 'event', '404_not_found', { 'event_category':'error', 'event_label':'page: ' + document.location.pathname + document.location.search + ' referrer: ' + document.referrer });";
 		}
 
 		$gtag_developer_id = '';
 		if ( ! empty( self::DEVELOPER_ID ) ) {
-			$gtag_developer_id = self::tracker_var() . "('set', 'developer_id." . self::DEVELOPER_ID . "', true);";
+			$gtag_developer_id = "gtag('set', 'developer_id." . self::DEVELOPER_ID . "', true);";
 		}
 
+		$gtag_name          = self::tracker_var();
 		$gtag_id            = self::get( 'ga_id' );
 		$gtag_cross_domains = ! empty( self::get( 'ga_linker_cross_domains' ) ) ? array_map( 'esc_js', explode( ',', self::get( 'ga_linker_cross_domains' ) ) ) : array();
-		$gtag_snippet       = '
+
+		$gtag_snippet = "(()=>{
 		window.dataLayer = window.dataLayer || [];
-		function ' . self::tracker_var() . '(){dataLayer.push(arguments);}
-		' . self::tracker_var() . "('js', new Date());
+		// Publish gtag under custom name to the global scope.
+		// Preserve the custom `arguments.callee.name`, for backward compatibility (not sure if even needed).
+		const gtag = window.$gtag_name = function $gtag_name(){dataLayer.push(arguments);}
```

Comment:
2. Or if the `tracker_var` was just to avoid conflicts with other plugins using the "gtag", and not to allow customization of something very specific. We can hide it in our own namespace/scope. Like
```suggestion
		window.wcgai.gtag = function gtag(){dataLayer.push(arguments);}
```
And use it as `wcgai.gtag( 'event', //...` everywhere else